pkg/virt-controller/watch/application.go

```diff
@@ -147,8 +148,16 @@ func (vca *VirtControllerApp) Run() {
 		golog.Fatalf("unable to get hostname: %v", err)
 	}
 
+	// TODO: Replace leaderelectionconfig.DefaultNamespace with a flag
+	namespace := leaderelectionconfig.DefaultNamespace
+	if data, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace"); err == nil {
```

Comment:
We should check if the error is `os.IsNotExist(err)`. If it is any other error we need to exit.